examples/seismic/tutorials/16_source_wavefield_reconstruction.ipynb

```diff
@@ -0,0 +1,1069 @@
+{
```

Comment:
<p>Might be good to have this presented as percentage of max wavefield value?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>